mil_common/drivers/mil_passive_sonar/src/sylphase_ros_bridge.cpp

```diff
@@ -109,7 +106,7 @@ void SylphaseSonarToRosNode::read_messages(boost::asio::ip::tcp::socket& socket)
       // Received packets are Big-Endian, convert to system
       for (size_t i = 0; i < msg.data.size(); ++i)
       {
-        msg.data[i] = ntohs(msg.data[i]) + SIGNED_TO_UNSIGNED;
+        msg.data[i] = ntohs(msg.data[i]);
```

Comment:
The `Ping.msg` is UNSIGNED whereas the data from sylphase is signed.... so I think this will cause problems as random signed intergers will be interpreted as unsigned on the receiving end.

So either:
* Revert this change
* Change the data type in `Ping.msg` to int16

Reply:
Good catch, Turns out this was causing some issues, Has been resolved as of Added HydrophoneSamples msg.